cmd/mps-control-daemon/mount/mount-shm.go

```diff
@@ -20,23 +20,55 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
+	"regexp"
 
 	"github.com/urfave/cli/v2"
 	"k8s.io/mount-utils"
 )
 
+type options struct {
+	shmSize string
+}
+
 // NewCommand constructs a mount command.
 func NewCommand() *cli.Command {
-	// Create the 'generate-cdi' command
-	return &cli.Command{
-		Name:   "mount-shm",
-		Usage:  "Set up the /dev/shm mount required by the MPS daemon",
-		Action: mountShm,
+	opts := options{}
+
+	c := cli.Command{
+		Name:  "mount-shm",
+		Usage: "Set up the /dev/shm mount required by the MPS daemon",
+		Before: func(ctx *cli.Context) error {
+			return validateFlags(ctx, &opts)
+		},
+		Action: func(ctx *cli.Context) error {
+			return mountShm(ctx, &opts)
+		},
```

Comment:
I has assumed we would do this as a configuration parameter in the config file

Reply:
Yes, I had that in mind too and was looking into that as a follow up to this commit.

Would you ONLY expose it in the config file?

Reply:
I think we definitely want it in the config -- I don't have a strong opinion about whether it is *also* a command line flag. Probably not though, since other MPS options aren't.